blackbox/docs/sql/statements/create-table.rst

```diff
@@ -519,4 +511,4 @@ the NGramTokenizer or the NGramTokenFilter. The default is 1.
 ....................
 
 Specifies the maximum difference between min_shingle_size and max_shingle_size
-when using the ShingleTokenFilter. The default is 3.
\ No newline at end of file
+when using the ShingleTokenFilter. The default is 3.
```

Comment:
Unrelated change 🤓

Reply:
it's some automatic trimming of trailing whitespace :-D